### DISCUSSION TITLE
Convert text file to png?

### DISCUSSION THREAD

**Discussion by valdomero777** [dlemstra/Magick.NET#1033] [Help]
How can I convert a text file into a png or other format on .net?

**Comment by dlemstra** [comment]
You could do something like this:

```C#
var width = 1000;
var height = 1000;
var data = File.ReadAllText("test.txt");
using (var image = new MagickImage("label:" + data, width, height))
{
    image.Write("test.png");
}
```

**Reply by valdomero777** [reply]
Thanks for the answer, I tried in my code but happened something weird, just read the first character and made the image just with that character, did you have some problem when you try? would you know some solution for that?

**Reply by dlemstra** [reply]
Sounds like an issue with your input file or that you are not reading all the content of the text file.
